src/screen-reader-announcements.ts

```diff
@@ -1,56 +1,19 @@
-const createActiveDescendantString = (activeDescendant: string) => {
-  return `Press Enter to select ${activeDescendant}.`
+export const createAutoselectString = (firstOption: string): string => {
```

Comment:
`export function createA... () {` would probably be a better syntax now.  Both work, but there isn't a lot of value in using consts here instead of functions

Reply:
That's true. We're not doing anything with `this`. I can use the `function` syntax.